src/components/views/avatars/BaseAvatar.tsx

```diff
@@ -46,6 +46,8 @@ interface IProps {
     inputRef?: React.RefObject<HTMLImageElement & HTMLSpanElement>;
     className?: string;
     tabIndex?: number;
+    altText?: string;
+    ariaLabel?: string;
 }
 
 const calculateUrls = (url, urls, lowBandwidth) => {
```

Comment:
Could these get types whilst you're here please

```suggestion
const calculateUrls = (url: string, urls: string[], lowBandwidth: boolean) => {
```